tests/test_consumer.py

```diff
@@ -116,6 +116,19 @@ async def test_simple_consumer(self):
         # will ignore, no exception expected
         await consumer.stop()
 
+    def test_create_producer_no_running_loop(self):
+        loop = asyncio.new_event_loop()
```

Comment:
Could we run this test in a separate thread or make sure we don't have any loop set up in the test?